azure/resource_groups/alerts/template.json

```diff
@@ -36,9 +36,6 @@
     },
     "databaseServerId": {
       "type": "string"
-    },
-    "keyVaultId": {
-      "type": "string"
```

Comment:
You'll need to also remove this from `bin/azure-deploy`.

Reply:
Thanks F, I've added a commit that I think removes it from the right place 👍